_assets/stylesheets/_base.scss

```diff
@@ -153,3 +153,11 @@ img {
   color: $white;
   background-color: $gray-darkest;
 }
+
+.text-center {
```

Comment:
I would prefer to not use this and add the properties to the right classes in modules/tags.

Reply:
In my experience, I have encountered the need to use text-center a lot, so I have this clreated for almost all of my projects. And sure, let me know what styling you end up with. I was never good at that part anyway.

Reply:
@xRahul I know but for now please keep it in one file. I will introduce some functionality classes for this later!

Reply:
Should I inline it till then? or another scss file that gets imported in light/dark?

Reply:
@xRahul I would just add the properties to the classes created in tags.scss to be honest!

Reply:
that's fine too. I'll update the pr